src/test/java/ro/unibuc/hello/dto/GreetingTest.java

```diff
@@ -5,15 +5,15 @@
 
 class GreetingTest {
 
-    Greeting myGreeting = new Greeting(1, "John");
+    Greeting myGreeting = new Greeting("1", "John");
```

Comment:
Not sure if this test pulls its weight, we can probably remove it in a separate PR.